### PR TITLE
Update the WebDAV file/search properties

### PR DESCRIPTION
**Pull request by settermjd**
This relates to #276, and fully documents the list of all WebDAV properties for file/search requests.

### CODE REVIEW

**Comment by settermjd** [comment]
@PVince81, I'm not sure if this is exactly what you're after, or if you want/need it to go much further than it currently is. Really need your feedback on this one, to ensure that it's implemented correctly.

File: modules/developer_manual/examples/core/webdav_api/search/request/search_files/search_body_with_properties.xml [hunk 1]
```diff
@@ -3,20 +3,22 @@
...
+        <a:getlastmodified />
+        <a:getetag />
+        <a:getcontenttype />
+        <a:resourcetype />
+        <oc:fileid />
+        <oc:permissions />
+        <oc:size />
+        <a:getcontentlength />
+        <oc:tags />
+        <oc:favourite />
```
Comment (PVince81) [inline modules/developer_manual/examples/core/webdav_api/search/request/search_files/search_body_with_properties.xml]: => "favorites"

**Comment by PVince81** [comment]
The list looks fine.

Please note that it also applies to regular PROPFINDs and is not limited to searches

**Comment by settermjd** [comment]
> The list looks fine.
> 
> Please note that it also applies to regular PROPFINDs and is not limited to searches

Thank you for the feedback, @PVince81.

**Comment by settermjd** [comment]
> The list looks fine.
> 
> Please note that it also applies to regular PROPFINDs and is not limited to searches

Thanks. I'll integrate the list in with PROPFINDs in a follow-up PR.

File: modules/developer_manual/examples/core/webdav_api/search/request/search_files/search_body_with_properties.xml [hunk 2]
```diff
@@ -3,20 +3,20 @@
...
-        <a:getlastmodified />
-        <a:getetag />
+        <a:getcontentlength />
         <a:getcontenttype />
+        <a:getetag />
+        <a:getlastmodified />
+        <a:lockdiscovery />
         <a:resourcetype />
+        <oc:comments-unread />
+        <oc:favourites />
```
Comment (PVince81) [inline modules/developer_manual/examples/core/webdav_api/search/request/search_files/search_body_with_properties.xml]: => favorites (en_US form)

Comment (settermjd) [reply modules/developer_manual/examples/core/webdav_api/search/request/search_files/search_body_with_properties.xml]: np. Will make that change.

Comment (settermjd) [reply modules/developer_manual/examples/core/webdav_api/search/request/search_files/search_body_with_properties.xml]: Done.

**Pull request merged**